src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -49,6 +133,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
 
     String username = (String) request.getSession().getAttribute("user");
 
+    List<User> users = userStore.getAllUsers();
+    String newestUser = users.get(users.size()-1).getName();
```

Comment:
These might have a dependency on how `getAllUsers` is defined. `getAllUsers` do not guarantee that the users will be sorted based on joinedDate. 

Consider what happens if there are millions of users, is there a possibility of getting the latest user without loading all users?